base64.js

```diff
@@ -37,7 +37,7 @@
   // [https://gist.github.com/1020396] by [https://github.com/atk]
   object.atob || (
   object.atob = function (input) {
-    var str = String(input).replace(/=+$/, '');
+    var str = String(input).replace(new RegExp("=+$"), ''); // #31: ExtendScript bad parse of /=
```

Comment:
Did you see @michaelficarra's suggestion in #31? It strikes me as a nice approach. Another option is `/[=]+$/` (I just tested it in ExtendScript Toolkit).

I'm happy with any of these three fixes. @michaelficarra, do you have a preference?